portal/db.py

```diff
@@ -114,6 +114,16 @@ def insert_users():
                         row
                     )
 
+            with open('./portal/data/sessions.csv', 'r') as f:
+                reader = csv.reader(f)
+                next(reader)
+
+                for row in reader:
+                    cur.execute(
```

Comment:
I don't think our user will want to have pre-created sessions or courses so let's get rid of this and the sessions.csv file as well as the courses.csv file.